crates/autopilot/src/boundary/order.rs

```diff
@@ -21,10 +22,12 @@ pub fn to_domain(
             amount: order.data.buy_amount.into(),
         },
         protocol_fees,
+        created_on: Some(order.metadata.creation_date),
```

Comment:
> Should we be consistent here?

I just tried to avoid additional conversion logic from/to plain numbers and to send it as it is. Will try to use `u32` instead.

Reply:
Renamed and replaced with `u32`.